### DISCUSSION TITLE
스터디 1주차 문제

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#37] [공지사항]
스터디 0주차(Apr 21 - Apr 27), 1주차(Apr 28 - May 4)에 푸셔야 할 문제를 안내드립니다.
0주차 후반부에 참여하시는 분들은 따라 오시기 벅찰 것 같아서 예외적으로 2주를 묶었습니다.

리트코드 문제를 풀어보신 경험이 없다는 분들이 계셔서 일부로 난이도가 Easy인 문제만을 선정을 하였습니다.
그럼에도 불구하고 리트코드 문제를 처음 풀어보시는 분들은 분명히 어렵다고 느끼실 거에요. (저도 옛날에 그랬거든요 ㅋㅋ 😝)

초보자 분들께 간단히 팁을 드리면 문제 당 1시간이나 2시간 정도 스스로 풀 수 있는 제한 시간을 두세요.
그 정도 안에 못 푸시면 풀이를 참고하셔서 해결을 하신 후에 다음 문제로 과감히 넘어가시기를 추천드립니다.
이렇게 스스로 문제를 푸는데 걸리는 시간을 제한하는 방식이 효과적인 이유에 대해서는 아래 글을 읽어보시면 좋을 것 같아요.
https://www.algodale.com/guides/can-i-see-solutions/

풀이를 보셔도 이해가 어려운 경우에는 (그럴 수 잇습니다), 아래 링크를 통해서 GitHub에 질문을 올려주시면 저를 포함한 다른 스터디 멤버들이 도움이 줄 것입니다.
https://github.com/DaleStudy/leetcode-study/discussions/categories/%EB%8F%84%EC%9B%80%EC%9A%94%EC%B2%AD

풀이/제출 방법은 아래 링크를 참조바라겠습니다.
https://github.com/DaleStudy/leetcode-study/blob/main/README.md

## Contains Duplicate

- 문제: https://leetcode.com/problems/contains-duplicate/
- 해설: https://www.algodale.com/problems/contains-duplicate/

## Valid Anagram

- 문제: https://leetcode.com/problems/valid-anagram/
- 해설: https://www.algodale.com/problems/valid-anagram/

## Two Sum

- 문제: https://leetcode.com/problems/two-sum/
- 해설: https://www.algodale.com/problems/two-sum/

## Valid Palindrome

- 문제: https://leetcode.com/problems/valid-palindrome/
- 해설: https://www.algodale.com/problems/valid-palindrome/

## Best Time to Buy and Sell Stock

- 문제: https://leetcode.com/problems/best-time-to-buy-and-sell-stock/
- 해설: https://www.algodale.com/problems/best-time-to-buy-and-sell-stock/

**Comment by DaleSeo** [comment]
문제 관련해서 질문있으시면 답글로 남겨주세용 😃

**Comment by bky373** [comment]
안녕하세요 @DaleSeo 님,
혹시 Week 0~1 기간을 알 수 있을까용?

**Comment by DaleSeo** [comment]
안녕하세요 @bky373 님, Apr 21 - May 4 입니다. 본문에 날짜 추가하였습니다.

**Comment by DaleSeo** [comment]
Valid Palindrome 문제에 대한 해설을 알고달레에 올려드렸으니 혼자 문제 푸시기 어려운 분들은 참고 바랍니다.
